nx/include/switch/services/set.h

```diff
@@ -23,6 +23,17 @@ typedef enum {
     ColorSetId_Dark  = 1,
 } ColorSetId;
 
+/// Console Product Models
+typedef enum {
+    ProductModel_INVALID = 0, ///< Invalid Model
```

Comment:
SetSysProductModel_Invalid, etc

Reply:
should I rename the enum type to SetSysProductModel as well?

Reply:
yes